src/cli.py

```diff
@@ -0,0 +1,129 @@
+from rashdf import RasGeomHdf
+
+import fiona
+from geopandas import GeoDataFrame
+
+import argparse
+from ast import literal_eval
+import sys
+from typing import List
+
+
+COMMANDS = [
+    "mesh_areas",
+    "mesh_cell_points",
+    "mesh_cell_polygons",
+    "mesh_cell_faces",
+    "refinement_regions",
+    "bc_lines",
+    "breaklines",
+    "structures",
+]
+
+
+def docstring_to_help(docstring: str) -> str:
```

Comment:
What if there is no docstring?

Reply:
@sray014 updated function and test to handle null docstrings.